test/grouping.jl

```diff
@@ -24,8 +25,13 @@ module TestGrouping
 
     byf = by(dt, :a, dt -> DataTable(bsum = sum(dt[:b])))
 
-    @test all(T -> T <: AbstractVector, map(typeof, colwise([sum], dt)))
-    @test all(T -> T <: Nullable, map(typeof, colwise(sum, dt)))
+    cw = colwise([sum], dt)
+    @test all(T -> T <: NullableVector, map(typeof, cw))
+    answer = [NullableArray([20]), NullableArray([12]), NullableArray([-0.4283098098931877])]
+    @test all(isequal(cw[i], answer[i]) for i in eachindex(cw, answer))
+    cw = colwise(sum, dt)
+    @test all(T -> isa(T, Nullable{DataType}), map(typeof, cw))
```

Comment:
I don't think that's what you meant here. Just `all(x -> isa(x, Nullable{Int}), cw)` should work. Could also use that approach to simplify the code above.